ibis/backends/pandas/trace.py

```diff
@@ -124,6 +124,10 @@ def trace(func):
 
     @functools.wraps(func)
     def traced_func(*args, **kwargs):
+        import ibis
```

Comment:
what are these changes?

Reply:
This fixes a bug in testing, where there's an ordering dependency that assumes that `import pandas` happens before executing certain operations. In backends that use the native pandas backend implicitly (like computing constants) this code path can be hit _without_ having run `import pandas` and so the configuration option is never registered and thus fails. This can be reproduced by running tests with `pytest-xdist` and setting `--numprocesses` to something larger than 1.

Reply:
Actually, what is `ibis.pandas` namespace now? I think pandas backend code is move to `ibis.backends.pandas` already

Reply:
`ibis.pandas` is a dynamic module attribute taken from the notion of an `entry_point`: https://github.com/ibis-project/ibis/blob/master/ibis/__init__.py#L56